scripts/cc-cli/src/interfaces/index.ts

```diff
@@ -0,0 +1,4 @@
+// Auto-generated via `yarn polkadot-types-from-defs`, do not edit
```

Comment:
Also consider this https://github.com/gluwa/creditcoin/blob/dev/.github/workflows/ci.yml#L325

we're automatically updating the interfaces via GitHub Actions. If we really need to keep the interfaces in two places that may present a problem.

Reply:
Yeah it would be good to deduplicate the generated code. 

Not sure the best way to do that, because last I recall polkadot JS's use of tsconfig paths to augment the type info didn't play well with packaging - that may have changed (or I may be misremembering) though.

Reply:
https://gluwa.atlassian.net/browse/CSUB-600